<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmyWidget</class>
 <widget class="QWidget" name="QmyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>430</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Demo3</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Slider和QColor</string>
        </property>
        <widget class="QLabel" name="red">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>  RED  </string>
         </property>
        </widget>
        <widget class="QLabel" name="green">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string> Green</string>
         </property>
        </widget>
        <widget class="QLabel" name="blue">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
        <widget class="QLabel" name="alpha">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>41</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string> Alpha</string>
         </property>
        </widget>
        <widget class="QSlider" name="redSlider">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>160</width>
           <height>19</height>
          </rect>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
        <widget class="QSlider" name="greenSlider">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>50</y>
           <width>160</width>
           <height>19</height>
          </rect>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="blueSlider">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>70</y>
           <width>160</width>
           <height>19</height>
          </rect>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QSlider" name="alphaSlider">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>90</y>
           <width>160</width>
           <height>19</height>
          </rect>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QTextEdit" name="textEdit">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>30</y>
           <width>91</width>
           <height>81</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>水平</string>
        </property>
        <widget class="QScrollBar" name="horizontalScrollBar">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>331</width>
           <height>20</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>331</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="format">
          <string>0</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>331</width>
           <height>19</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Dial和LCDNumber</string>
        </property>
        <widget class="QDial" name="dial">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>115</width>
           <height>115</height>
          </rect>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>115</width>
           <height>111</height>
          </rect>
         </property>
        </widget>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>91</width>
           <height>111</height>
          </rect>
         </property>
         <property name="title">
          <string>LCD显示进制</string>
         </property>
         <widget class="QRadioButton" name="radioDec">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>89</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>十进制</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioBin">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>89</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>二进制</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioOct">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>89</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>八进制</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioHex">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>89</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>十六进制</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridRight">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>垂直</string>
        </property>
        <widget class="QProgressBar" name="progressBar_2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>30</y>
           <width>16</width>
           <height>241</height>
          </rect>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QSlider" name="verticalSlider">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>19</width>
           <height>241</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QScrollBar" name="verticalScrollBar">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>16</width>
           <height>241</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
